.eslintrc.json

```diff
@@ -44,7 +44,11 @@
     "react/jsx-props-no-spreading": "off",
     "no-useless-constructor": "off",
     "@typescript-eslint/triple-slash-reference": "off",
-    "@typescript-eslint/no-unused-vars": "off"
+    "@typescript-eslint/no-unused-vars": "off",
+    "class-methods-use-this": "off",
+    "no-debugger": "off",
+    "import/order": "off",
+    "no-console": "off"
```

Comment:
@alicjak1519 
Jeśli będą pojawiać się jakieś dziwne warningi które twoim zdaniem nie będą mieć sensu, to możesz wyłączać je w tym pliku w taki sposób jak wyżej